db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_05_28_204239) do
+ActiveRecord::Schema[7.0].define(version: 2022_05_31_073016) do
```

Comment:
no changes? strange... have you forgot anything?

Reply:
That was the free migration for the recovery password that already I had  😅